torch/_inductor/kernel/mm_scaled.py

```diff
@@ -5,8 +5,10 @@
 
 import torch
 from torch._inductor.codegen.rocm.ck_universal_gemm_template import CKGemmTemplate
+from torch.utils._triton import has_triton_tma
```

Comment:
@drisspg this one checks for existence of the host-side TMA API (i.e., `create_1d_tma_descriptor` and `create_2d_tma_descriptor`). Here you need device-side TMA API which seems to have been added in a later PR https://github.com/triton-lang/triton/pull/4633. I guess, we need another helper function checking if `experimental_device_tensormap_create1d` and `experimental_device_tensormap_create2d` are available?

Reply:
will do